tools/BUILD.bazel

```diff
@@ -12,6 +14,21 @@ config_setting(
     },
 )
 
+# Bazel build setting for enabling partial linker compilation. The partial linker compilation
+# build setting can be set using `--//tools:partial_linker_compilation=True`.
+bool_flag(
+    name = "partial_linker_compilation",
```

Comment:
We should avoid using the term `linker` on this side of the compilation. All that is happening is the compiler is producing "partial Ivy" compiled code.  The linking is done on the downstream side, which is not affected by this flag AFAICT.